lib/src/json_schema/constants.dart

```diff
@@ -155,8 +155,12 @@ class SchemaVersion implements Comparable<SchemaVersion> {
         return draft7;
       case 'https://json-schema.org/draft/2019-09/schema':
         return draft2019_09;
+      case 'https://json-schema.org/draft/2019-09/schema#':
```

Comment:
It's weird that this is the fix. I was using the no-anchor version, but it's probably getting appended somewhere.

Reply:
Yes, it is getting appended somewhere. There should probably be a refactor to bring some consistency to how the URLs are being handled, but I think that's outside the scope of this fix.